pages/docs/manual/latest/pipe-last.mdx

```diff
@@ -0,0 +1,51 @@
+# Pipe Last
+
+<Warn>
+We discourage the usage of the `|>` / pipe-last operator in BuckleScript codebases. Using it may sometimes have a runtime cost due to currying.
+Please use the [pipe-first](./pipe-first) operator instead.
+
+Pipe-last is mainly used in native / OCaml codebases or in functional programming libraries
+aiming for a `data last` approach. You'll also find it in older Reason codebases and libraries.
+</Warn>
+
+[`|>`](https://reasonml.github.io/api/Pervasives.html#VAL(|%3E)) is an infix operator for piping a value to the last argument position of a function.
+
+**Example:**
+
+```reason
+let personSays = (person, msg) => person ++ " says: " ++ msg;
+
+"This will be put in the msg arg"
+  |> personSays("Alpaca");
+
+/* Returns: "Alpaca says: This will be put in the msg arg */
+```
+
+### Differences between pipe-last and pipe-first
+
+- Pipe-last is a **function defined as an infix operator**. Pipe-first (`->`) is implemented as syntactic sugar within BuckleScript (no runtime function call involved)
```

Comment:
```suggestion
- Pipe-last is a **binding to a compiler intrinsic,** it will optimize some currying, but not all. Pipe-first (`->`) is implemented as syntactic sugar within BuckleScript and guarantees zero extra currying overhead.
```

Reply:
suggestion accepted